### ISSUE TITLE
Purified Persona Pics on main NFT screen is not indicative of their PURE status.

### ISSUE THREAD

**Issue by k1l1eN** [vessenes/persona#8]
The picture on the main NFT screen for Purified persona do not light up like they do in the collective or "other" tabs.  As a suggestion you keep the indicators of being "pure" unanimous across the board.  IE.  Add the lit up feature wherever the pic is shown.

<img width="1076" alt="image" src="https://github.com/user-attachments/assets/b79cc173-ff31-4ec7-ae32-d465be1c0ddc">

<img width="407" alt="image" src="https://github.com/user-attachments/assets/4a2f15a0-b6b1-472a-9370-021240cfcc96">

**Comment by vessenes** [comment]
Yes, this is a problem. In fact, they do get brightened up. But, I am having an immense amount of trouble getting the glow to extend beyond the encapsulating div. I would welcome any CSS fixes from a front-end person; I'm at my wit's end about it.
